library/std/src/sys/pal/unix/thread.rs

```diff
@@ -803,12 +803,30 @@ pub mod guard {
         Some(stack_ptr.with_addr(stackaddr))
     }
 
+    #[cfg(target_os = "netbsd")]
+    unsafe fn get_stack_start() -> Option<*mut libc::c_void> {
+        let mut ret = None;
+        let mut attr: libc::pthread_attr_t = crate::mem::zeroed();
+        let e = libc::pthread_getattr_np(libc::pthread_self(), &mut attr);
```

Comment:
Maybe this should note the specific reason that we're choosing `getattr_np` here instead of `pthread_attr_get_np`, since as I understand it, that relates directly to the validity question above.

Reply:
Hm? I did not say it looked not-fine, I said it could use a comment explaining this.